src/components/MessageFormRtc.js

```diff
@@ -0,0 +1,104 @@
+/* eslint-disable eqeqeq */
+/* eslint-disable consistent-return */
+/* eslint-disable dot-notation */
+/* eslint-disable no-unused-vars */
+/* eslint-disable react/prop-types */
+/* eslint-disable react/jsx-no-bind */
+import React, { useState, useEffect } from 'react'
+import { connect } from 'react-redux'
+import { closeWebRtc, setRtcUserId, connectWebRtc } from '../actions/rtc'
+import styles from '../styles/MessageForm.module.css'
+import ChatHeader from './ChatHeader'
+import FormInputRtc from './FormInputRtc'
+import MessageBubble from './MessageBubble'
+
+const MessageFormRtc = ({ selected, messages, userId, connection, chatInfo, connWRtc, closeWRtc, setRtcUid }) => {
+  const [dragActiveState, setDragActiveState] = useState(false)
+  const [filesDrag, setFilesDrag] = useState(null)
+  const [chatMessages, setChatMessages] = useState([])
+
+  useEffect(() => {
+    if (connection) {
+      connWRtc(connection)
+    } else {
+      setRtcUid()
+    }
+  }, [connWRtc, closeWRtc, setRtcUid, selected, userId, messages, connection])
+
+  const msgList = []
+
+  const resultEnd = React.useRef(null)
+
+  if (typeof messages !== 'undefined' && messages !== null && messages.length > 0) {
+    let i = 0
+    messages.forEach((msg) => {
+      let bubble = <MessageBubble className={styles.message_bubble.them} key={i} msgInfo={msg} />
+      if (msg.self) {
+        bubble = <MessageBubble className={styles.message_bubble.me} key={i} msgInfo={msg} />
```

Comment:
Что если попробовать вот так:
```suggestion
      const bubble = <MessageBubble className={styles.message_bubble[msg.self ? 'me' : 'them']} key={i} msgInfo={msg} />
```